File: Basic Math/CountOddDigits/code.java
class Solution {
    public int countOddDigit(int n) {
        int count = 0;
        if (n == 0) return 0;
        while (n != 0){
            if ((n % 10 ) % 2 != 0) count++;
            n /= 10;
        }
        return count;
    }
}

File: Basics/Basic Math/Count of Prime Numbers till N/code.py
class Solution:
    def primeUptoN(self, n):
        count = 0
        for i in range(2, n + 1):
            for j in range(2, i):
                if i % j == 0:
                    continue
                else:
                    count += 1
                    break
        return count - 1
